docs/Client_Libraries/go.mdx

```diff
@@ -182,3 +176,68 @@ func main() {
     }
 }
 ```
+
+### Pull a docker or OCI image from a remote repository
+You can also use an OCI layout folder to store docker/OCI images:
+```golang

```

Comment:
```suggestion

```go

```

Reply:
You need to add one line before the code block then it will take a highlight effect on the website.
You can preview the changes at https://deploy-preview-143--oras-project.netlify.app/docs/Client_Libraries/go#pull-a-docker-or-oci-image-from-a-remote-repository